backend/infrahub/graphql/mutations/computed_attribute.py

```diff
@@ -92,18 +92,20 @@ async def mutate(
             )
         if attribute_field.value != str(data.value):
             attribute_field.value = str(data.value)
-            await target_node.save(db=context.db, fields=[str(data.attribute)])
+            async with context.db.start_transaction() as dbt:
+                await target_node.save(db=dbt, fields=[str(data.attribute)])
 
-            log_data = get_log_data()
-            request_id = log_data.get("request_id", "")
+                log_data = get_log_data()
+                request_id = log_data.get("request_id", "")
 
-            graphql_payload = await target_node.to_graphql(db=context.db, filter_sensitive=True)
+                graphql_payload = await target_node.to_graphql(db=dbt, filter_sensitive=True)
```

Comment:
not sure if this matters here, but I have encountered problems where changes are not written to the database until the transaction is closed and that prevents us from getting the latest changes.
so maybe this `to_graphql` should be outside of the transaction

Reply:
Ok! I moved this part out of the transaction.